frigate/plus.py

```diff
@@ -17,45 +21,54 @@ def get_jpg_bytes(image, max_dim, quality):
     original = cv2.resize(image, dsize=(width, height), interpolation=cv2.INTER_AREA)
 
     ret, jpg = cv2.imencode(".jpg", original, [int(cv2.IMWRITE_JPEG_QUALITY), quality])
-    return jpg.tobytes()
+    jpg_bytes = jpg.tobytes()
+    return jpg_bytes if type(jpg_bytes) is bytes else b""
 
 
 class PlusApi:
-    def __init__(self, host, key: str):
```

Comment:
There is (currently) only one instance of this class (obviously :) 
I decided to make all arguments non-dynamic.